Assets/com.gdcbd.bossbattle/Scripts/Components/ProjectileController.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using DG.Tweening;
+using UnityEngine;
+
+namespace com.gdcbd.bossbattle.components
+{
+    [CreateAssetMenu(fileName = "NewProjectileController", menuName = "BossBattle/ProjectileController")]
+    public class ProjectileController : AbstractProjectileController
+    {
+        [SerializeField] private GameObject _projectilePrefab;
+        [SerializeField] private float _speed = 200f;
+        [SerializeField] private float _damage;
+        [SerializeField] private float _lifeSpan = 2f;
+       
+        
+       
+        public override void Launch(ShootInfo shootInfo)
```

Comment:
The method should be named Shoot. As the variable itself passed is called shoot info.

Reply:
Can not solve, 
here Gun already Shoot with with this ShootInfo
and this is PrejectileController, Projectile should Launch with the Shoot Information

Reply:
What I meant was rename Launch Method as `Shoot(ShootInfo shootInfo)` Like this